fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/CloudGlobalTransactionMgr.java

```diff
@@ -346,7 +346,29 @@ public void commitTransaction(long dbId, List<Table> tableList,
     public void commitTransaction(long dbId, List<Table> tableList, long transactionId,
             List<TabletCommitInfo> tabletCommitInfos, TxnCommitAttachment txnCommitAttachment)
             throws UserException {
-        commitTransaction(dbId, tableList, transactionId, tabletCommitInfos, txnCommitAttachment, false);
+        List<OlapTable> mowTableList = getMowTableList(tableList, tabletCommitInfos);
+        try {
+            LOG.info("try to commit transaction, transactionId: {}", transactionId);
```

Comment:
Is the log ncessary?

Reply:
this log record when current transaction successfully getting fe's commitLock and start do get delete bitmap lock, it's useful for checking problem